vnext/local-cli/generator-windows/templates/cpp/src/MainPage.h

```diff
@@ -0,0 +1,21 @@
+﻿#pragma once
+#include "MainPage.g.h"
+#include <winrt/Microsoft.ReactNative.h>
+
+// clang-format off
+namespace winrt::<%=ns%>::implementation
```

Comment:
><%=ns%> [](start = 17, length = 7)

I wonder how it works: namespace here must have '::' separator, while in xaml file the '.' separator.
Do we support complex namespaces?

Reply:
we set this to be either the name of the project, or what's passed as the `--ns` option:
```js
  const ns = options.ns || newProjectName;
```